libraries/Microsoft.Bot.Builder.Dialogs.Debugging/DebuggerSourceMap.cs

```diff
@@ -300,31 +304,5 @@ private IReadOnlyList<Protocol.Breakpoint> Update()
                 return changes;
             }
         }
-
-#pragma warning disable CA1034 // Nested types should not be visible (we can't change this without breaking binary compat, consider fixing this before we go out of preview)
-        public sealed class Row
```

Comment:
why not internal?  Why not standalone class?

Reply:
done